ngraph/test/onnx/onnx_import_controlflow.in.cpp

```diff
@@ -330,9 +330,6 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_controlflow_loop_the_proper_opset_in_subgraph)
         });
     const auto body_mul_node = ngraph::as_type_ptr<default_opset::Multiply>(*body_mul_node_it);
     EXPECT_TRUE(body_mul_node);
```

Comment:
Can we add tests to cover new behavior?

Reply:
We have some tests for that already. Take a look at onnx_model_mul_v6, onnx_model_add_v6, etc.